darshan-util/pydarshan/darshan/__init__.py

```diff
@@ -3,7 +3,7 @@
 PyDarshan also provides a suite of analysis utilities.
 """
 
-__version__ = '3.3.1.0'
+__version__ = '3.3.1.1'
```

Comment:
I just scanned our release checklist to remind myself if anything was forgotten here, and the only thing probably worth updating as part of this PR is the pydarshan changelog? Maybe indicate what's changed in our wheel support and what what was fixed in Darshan log discovery?

I'll run tests and also double check with NERSC that this resolves issues there (probably using test release on PyPI that @tylerjereddy mentions).

Reply:
NERSC confirmed that the new wheels work for them in https://github.com/darshan-hpc/darshan/issues/656#issuecomment-1045502017